scripts/install-dotnet.ps1

```diff
@@ -1,6 +1,7 @@
 ﻿#
-# Copyright (c) .NET Foundation and contributors. All rights reserved.
-# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+# Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This file was borrowed from https://github.com/dotnet/cli/blob/master/scripts/obtain/dotnet-install.ps1, so I am not sure about the license here.